api.md

```diff
@@ -2654,66 +2656,142 @@ SELECT * FROM timescaledb_information.data_node;
+(2 rows)
 ```
 
-If you want to see the current interval length for your hypertables, you can
-check the `_timescaledb_catalog` as follows. Note that for time-based interval
-lengths, these are reported in microseconds.
+## timescaledb_information.chunks [](timescaledb_information-chunks)
+
+Get metadata about the chunks of hypertables.
+This view shows metadata for the chunk's primary dimension.
+range_start_integer and range_end_integer are set if the type of the primary dimension type is integer based.
+range_start and range_end are set if the primary dimension is of time datatype.
+
+#### Available Columns [](timescaledb_information-chunks-available-columns)
+
+|Name|Description|
+|---|---|
+| `hypertable_schema` | Schema name of the hypertable |
+| `hypertable_name` | Table name of the hypertable |
+| `chunk_schema` | Schema name of the chunk |
+| `chunk_name` | Name of the chunk |
+| `primary_dimension` | Name of the column that is the primary dimension|
+| `primary_dimension_type` | Type of the column that is the primary dimension|
+| `range_start` | Start of the range for the chunk's dimension |
+| `range_end` | End of the range for the chunk's dimension |
+| `range_start_integer` | Start of the range for the chunk's dimension, if the dimension type is integer based |
+| `range_end_integer` | End of the range for the chunk's dimension, if the dimension type is integer based. |
+| `is_compressed` | Is the data in the chunk compressed?|
+| `chunk_tablespace` | Tablespace used by the chunk|
+| `data_nodes` | Nodes on which the chunk is replicated. This is applicable only to chunks for distributed hypertables. |
+
+#### Sample Usage [](timescaledb_information-chunks-examples)
+
+Get information about the chunks of a hypertable.
 
 ```sql
-SELECT h.table_name, c.interval_length FROM _timescaledb_catalog.dimension c
-  JOIN _timescaledb_catalog.hypertable h ON h.id = c.hypertable_id;
+CREATE TABLESPACE tablespace1 location '/usr/local/pgsql/data1';
+  
+CREATE TABLE hyper_int (a_col integer, b_col integer, c integer);
+SELECT table_name from create_hypertable('hyper_int', 'a_col', chunk_time_interval=> 10);
+CREATE OR REPLACE FUNCTION integer_now_hyper_int() returns int LANGUAGE SQL STABLE as $$ SELECT coalesce(max(a_col), 0) FROM hyper_int $$;
+SELECT set_integer_now_func('hyper_int', 'integer_now_hyper_int');
 
-table_name | interval_length
-------------+-----------------
-metrics       |    604800000000
-(1 row)
-```
+INSERT INTO hyper_int SELECT generate_series(1,5,1), 10, 50;
+
+SELECT attach_tablespace('tablespace1', 'hyper_int');
+INSERT INTO hyper_int VALUES( 25 , 14 , 20), ( 25, 15, 20), (25, 16, 20);
 
+SELECT * FROM timescaledb_information.chunks WHERE hypertable_name = 'hyper_int';
+
+ hypertable_schema | hypertable_name |     chunk_schema      |    chunk_name     | primary_di
```

Comment:
Maybe switch to extended view `\x` for these longer outputs?

Reply:
fixed.